### PR TITLE
Initial commit for card account

### PR DESCRIPTION
**Pull request by jnmtwelve**
Initial structure of card account. Need to implement offer submodule prior to completing cardaccount

### CODE REVIEW

File: src/main/java/fi/coronado/cardaccount/CardAccount.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import fi.coronado.TripleObject;
+import fi.coronado.baseobjects.BaseObjects;
+
+import java.util.*;
+
+public class CardAccount extends TripleObject {
+    /*Account status object.
```
Comment (kinabalu) [inline src/main/java/fi/coronado/cardaccount/CardAccount.java:13]: this should technically be above the class definition, in Javadoc format. nit, i wouldn't hold up an approval but let's try to fix this prior to merge

File: src/main/java/fi/coronado/selfie/Selfie.java [hunk 2]
```diff
@@ -4,10 +4,7 @@
 import fi.coronado.auth.Auth;
 import fi.coronado.config.Config;
 
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (kinabalu) [inline src/main/java/fi/coronado/selfie/Selfie.java:7]: I'm sure this was probably something IDEA did? I would actually prefer if we specified only the classes we use, no .*

**Review by kinabalu — APPROVED** [review]
i think some of the getter/setter stuff can be simplified by us integrating lombok into the library. https://projectlombok.org/features/GetterSetter

a nit on doing proper javadoc for CardAccountStatus and any other, should be above the class always.

File: src/main/java/fi/coronado/cardaccount/CardAccount.java (hunk 1 above)
Comment (jnmtwelve) [reply src/main/java/fi/coronado/cardaccount/CardAccount.java:13]: That java doc was for the enum class. Moot point now, as the enum got refactored out as part of the work on offer

File: src/main/java/fi/coronado/selfie/Selfie.java (hunk 2 above)
Comment (jnmtwelve) [reply src/main/java/fi/coronado/selfie/Selfie.java:7]: Yep! Sneaky IDE, I'll revert

**Pull request merged**